backend/app/controllers/api/v1/invitations_controller.rb

```diff
@@ -0,0 +1,43 @@
+module API
+  module V1
+    class InvitationsController < BaseController
+      before_action :authenticate_user!, except: :update
+      before_action :fetch_user_by_email, only: :create
+      before_action :fetch_user_by_token, only: :update
+      before_action :require_light_account!
+
+      def create
+        authorize! :invite, @user
+
+        @user.invite! current_user
+        head :ok
+      end
+
+      def update
+        @user.with_lock do
+          @user.assign_attributes params.permit(:password, :first_name, :last_name, :ui_language)
+          @user.assign_attributes @user.invited_by&.attributes&.slice("role", "account_id") if @user.invited_by.present?
```

Comment:
I would probably raise an error if a user that invites another user to the platform does not exist.

Reply:
To create this view 
![image](https://user-images.githubusercontent.com/1286444/173810279-2afaaa48-a082-427c-9f40-2a5359b021e8.png)

Reply:
Yeah, like this should not happen to be honest, probably put these `&` there during testing, will get rid of it ;)

Reply:
hmm, good point, will move it to create action

Reply:
It will be a little different than in the flow (we can update that file later), but I think it makes more sense from the future API `/account/users` API perspective.

Reply:
Ok, then let's leave it if it will be fine.